test/api/unit/models/build.test.js

```diff
@@ -52,54 +53,74 @@ describe('Build model', () => {
   });
 
   describe('after create hook', () => {
-    it('should send a build new build message', (done) => {
-      factory.build()
-        .then(build => build.updateJobStatus({ status: 'complete' }))
-        .then((build) => {
-          const queuedBuild = sendMessageStub.getCall(0).args[0];
-          const buildCount = sendMessageStub.getCall(0).args[1];
+    it('should send a build new build message', async () => {
+      const startedAt = new Date();
+      let build = await factory.build();
```

Comment:
prefer `const` over `let` unless the variable will be reassigned, I think linting should catch this...